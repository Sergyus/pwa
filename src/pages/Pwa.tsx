import React from 'react';
import Title from '../components/Title';
import pic from './../assets/img/pwa.jpg';
import pic2 from './../assets/img/pwa2.jpg';
import pic3 from './../assets/img/pwa3.jpg';

export default function Pwa(): JSX.Element {
  return (
    <>
      <Title title="PWA" />
      <img style={{ width: '100%' }} src={pic} alt="pic" />
      <br />
      <br />
      <img style={{ width: '100%' }} src={pic2} alt="pic" />
      <br />
      <br />
      <img style={{ width: '100%' }} src={pic3} alt="pic" />
    </>
  );
}
