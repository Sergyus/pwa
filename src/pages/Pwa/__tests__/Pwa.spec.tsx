import React from 'react';
import { render } from '@testing-library/react';
import Pwa from '../Pwa';

describe('<Pwa/>', () => {
  it('should renders', () => {
    const { asFragment } = render(<Pwa />);
    expect(asFragment()).toMatchSnapshot();
  });
});
