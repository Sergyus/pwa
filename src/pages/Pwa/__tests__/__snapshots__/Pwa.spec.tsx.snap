// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pwa/> should renders 1`] = `
<DocumentFragment>
  <h1
    class="title"
  >
    PWA
  </h1>
  <img
    alt="pic"
    src="IMAGE_MOCK"
    style="width: 100%;"
  />
  <br />
  <br />
  <img
    alt="pic"
    src="IMAGE_MOCK"
    style="width: 100%;"
  />
  <br />
  <br />
  <img
    alt="pic"
    src="IMAGE_MOCK"
    style="width: 100%;"
  />
</DocumentFragment>
`;
